armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -594,6 +596,24 @@ def test_transferIncompatibleStationaryBlocks(self):
         with self.assertRaises(ValueError):
             fh._transferStationaryBlocks(a1, a2)
 
+        # re-initialize assemblies
+        self.setUp()
```

Comment:
Calling `self.setUp` suggests that the assertions below would be very well suited for a new test! I noticed that `self.setUp()` is called a few times in this test file. It would be a welcome addition if you could split off each of those into individual tests! 

If it ends up being more work/a pain to do, we can create an issue and punt. But we should definitely split off your newest addition into its own test.

Reply:
I split up the test into three separate tests!

Reply:
Looks like your unit tests are failing as the new tests that got split off don't have a `fuelHandler` object. I think if you add `fh = fuelHandlers.FuelHandler(self.o)` to the tests, they should fall into place.

Reply:
Fixed!